ckanext/datatablesview/templates/datatables/datatables_view.html

```diff
@@ -40,7 +40,14 @@
         {% for field in [{'id': '_id'}] + h.datastore_dictionary(resource.id) %}
           {% if 'show_fields' not in resource_view
             or field.id in resource_view.show_fields %}
-            <th scope="col">{{ field.id }}</th>
+            <th scope="col">
+              {% set label = 'label_' + h.lang() %}
+              {% if label in field.info and field.info[label]|length %}
```

Comment:
```suggestion
              {% if label in field.info and field.info[label] %}
```
empty strings are already considered to be "false"